web/main/models.py

```diff
@@ -1653,6 +1656,41 @@ def export_content(self, request):
             return rich_text_export(self.resource.content, request=request, id_prefix=str(self.id))
         return self.resource.content
 
+    def get_ali_license_text(self):
+        licensed_materials = [
+            {
+                "title": "Restatements of the Law",
+                "copyright_year": 2014,
+                "match_words": [
+                    "restatement",
+                    "restatements",
+                    "contracts",
+                    "restatements of the law",
+                ],
+            },
+            {
+                "title": "Principles of the Law",
+                "copyright_year": 2015,
+                "match_words": ["principle", "principles", "principles of the law"],
+            },
+            {
+                "title": "Model Penal Code",
+                "copyright_year": 2016,
+                "match_words": ["model penal code", "mpc"],
+            },
+        ]
+
+        license_txt = ""
+        title = self.title.lower()
+
+        for item in licensed_materials:
+            if any(word.lower() in title.split() for word in item["match_words"]):
```

Comment:
I think this needs to compare phrases, not words, as `match_words` contains both; that is, I don't think we can use `title.split()`.

Reply:
I removed the split() since the words in match words are both single words and the phrases anyway and `if any(word.lower() in title...` will return True for both. Also, the actual list data hasn't been finalized yet as I noted in the PR description.